efel/cppcore/cfeature.cpp

```diff
@@ -374,12 +374,7 @@ int cFeature::calc_features(const string& name) {
   map<string, vector<featureStringPair> >::const_iterator lookup_it(
       fptrlookup.find(name));
   if (lookup_it == fptrlookup.end()) {
-    fprintf(stderr,
-            "\nFeature [ %s ] dependency file entry or pointer table entry is "
-            "missing. Exiting\n",
-            name.c_str());
-    fflush(stderr);
-    exit(1);
+      throw std::runtime_error("Feature dependency file entry or pointer table entry is missing.");
```

Comment:
I think you missed the ``name`` here

Reply:
Thanks, I just added it in the next commit :+1: